### DISCUSSION TITLE
Member Access

### DISCUSSION THREAD

**Discussion by felixsteinke** [hft-rcpsp-scheduling/.github#2] [Permissions]
Hello there, 

If you want to join the Group, please ask here and choose a preferred team.

* AI
* Heuristics
* Research

**Comment by mrvxk** [comment]
I want to join Heuristics please. Thank you in advice! ❤️

**Comment by mrclhng** [comment]
I want to join Heuristics please. Thank you in advice! ❤️

**Comment by Tiberiu07** [comment]
I would like to join heuristics 👍

**Comment by nikojochim** [comment]
I would like to join heuristics

**Comment by ArtemBaranovsky** [comment]
I want to join Research group please. Thanks for work you've done!

**Reply by mrvxk** [reply]
Would you like to research for AI group?

**Reply by Tiberiu07** [reply]
Hello Artem. There was a discussion within today's class and basically we came to the conclusion that research will be done within AI / Heuristics team.

**Reply by ArtemBaranovsky** [reply]
Hello, Tiberiu and Marvin!
In this case I must join to AI or Heuristic team.

**Reply by felixsteinke** [reply]
You can do research. Most parts will be probably for the support of the AI team, but we will discuss that on monday.

**Comment by harshalsojitra** [comment]
I want to be a part of AI / NL Team.

**Reply by felixsteinke** [reply]
I send an invite to you. You have to accept it to be in the group.

**Comment by chptlouis** [comment]
Hi, I would like to be in heuristics team, thanks !

**Reply by felixsteinke** [reply]
I send an invite to you. You have to accept it to be in the group.

**Comment by pradeephiremath** [comment]
I would like to be in AI team, please

**Comment by ghost** [comment]
Hi Felix,
I would like to join the AI team.

Thanks :)

**Comment by homberger18** [comment]
hi
